docs/index.html

```diff
@@ -12,16 +12,7 @@
             <h1>MA Projects</h1>
             <ul>
                 <li>
-                    Vladyslav Yermolin - <a href="./vladyslav.yermolin/index.html">Lecture 5</a>
-                </li>
-                <li>
-                    Serhii Noskov - <a href="./SerhiiNSN/index.html">Final Project</a>
-                </li>
-                <li class="allocene">
-                    Artur Kobyliatsky - <a href="./Allocene/index.html">Final Project</a>
-                </li>
-                <li>
-                    Ihor Zapisochniy - <a href="./IgorFromUA/index.html">Final project</a>
+                    Lesia Turuta - <a href="./lesiaturuta/PokemonProject/login.html">Pokemons</a>
```

Comment:
Hey, @lesiaturuta, please don't **delete** links to your colleagues' projects - just **add** your link